src/reducers/todos.js

```diff
@@ -0,0 +1,59 @@
+const todos = (
+  state = {
```

Comment:
Its better to give default values in each switch case. Otherwise as your app will grow you will have loads of default values. Use Destructuring and default values, you will be able to do it.

Reply:
Okay. Will change that